### DISCUSSION TITLE
How to Enable Shaders

### DISCUSSION THREAD

**Discussion by Charlielx** [jellyfin/jellyfin-mpv-shim#368] [Q&A]
So I'm not sure if I'm missing something or it's just that I'm dumb lol but I can't seem to figure out how to actual enable shaders for the life of me. What specifically do you have to do to enable Anime4k for example or just filter through the available options?

**Comment by SinTan1729** [comment]
In case you're using the package from Arch repos, there's some kind of issues. I switched to the `pip` package, and it works perfectly. I installed it using `uv`.
```
uv tool install jellyfin-mpv-shim --with pillow --with pystray
```
